OpenEdXMobile/res/layout/dialog_upgrade_features.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+        <LinearLayout
+            android:id="@+id/dialog_layout"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@android:color/white"
+            android:orientation="vertical"
+            android:padding="@dimen/dialog_padding_top_material">
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:id="@+id/dialog_dismiss"
+                android:layout_width="@dimen/fa_x_large"
+                android:layout_height="@dimen/fa_x_large"
+                android:layout_gravity="end"
+                android:src="@drawable/ic_close"
+                android:tint="@color/primaryBaseColor" />
+
+            <TextView
+                android:id="@+id/dialog_title"
+                style="@style/semi_bold_primary_base_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="20dp"
+                android:gravity="center"
+                android:textSize="@dimen/edx_x_large"
+                tools:text="@string/course_dashboard_modal_title" />
+
+            <androidx.appcompat.widget.AppCompatImageView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="center_horizontal"
+                android:layout_margin="@dimen/container_padding"
+                android:src="@drawable/course_certificate" />
+
+            <TextView
+                style="@style/regular_primary_base_text"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="20dp"
+                android:gravity="start"
+                android:text="@string/course_modal_heading"
+                android:textSize="@dimen/edx_large" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="20dp"
+                android:orientation="horizontal">
+
+                <androidx.appcompat.widget.AppCompatImageView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:padding="@dimen/widget_margin_x_half"
+                    android:src="@drawable/ic_check_circle" />
+
+                <TextView
+                    style="@style/regular_primary_base_text"
```

Comment:
better to add some start margin to make parity with mock-ups.  around `5dp`.

same with the other textviews.

Reply:
Added